maintainer/release.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
Maintainer scripts are sooooo nineties... :wink: 

Can't the CI do this for us?

Reply:
no time!  :-)

also, i'm not going to give my haddocks credentials away, and i'm not sure i want to give github haddocks credentials at all.

Reply:
but yes, would be nice!